core/dbt/adapters/base/impl.py

```diff
@@ -781,14 +782,39 @@ def get_catalog(self, manifest):
         Returns an agate.Table of catalog information.
         """
         # make it a list so macros can index into it.
-        context = {'databases': list(manifest.get_used_databases())}
+        context = {
+            'databases': list(manifest.get_used_databases()),
```

Comment:
seems like this should be removed

Reply:
I don't think we can do that! All of the `get_catalog` macros in dbt loop over this value. The alternative would be to make the catalog queries loop over the `information_schema` dict values... unsure if that's better